docs/csharp/language-reference/builtin-types/struct.md

```diff
@@ -0,0 +1,65 @@
+
+Typically, you use structure types to design small data-centric types that provide little or no behavior. For example, .NET uses structure types to represent a number (both [integer](integral-numeric-types.md) and [real](floating-point-numeric-types.md)), a [Boolean value](bool.md), a [Unicode character](char.md), a [time instance](xref:System.DateTime). If you're focused on the behavior of a type, consider defining a [class](../keywords/class.md). Class types have *reference semantics*. That is, a variable of a class type contains a reference to an instance of the type, not the instance itself.
+
+## Limitations with the design of a structure type
+
+When you design a structure type, you have the same capabilities as with a [class](../keywords/class.md) type, with the following exceptions:
+
+- You cannot declare a parameterless constructor. Every structure type already provides an implicit parameterless constructor that produces the [default value](default-values.md) of the type.
+
+- You cannot initialize an instance field or property at its declaration. However, you can initialize a [static](../keywords/static.md) or [const](../keywords/const.md) field or a static property at its declaration.
+
+- A constructor of a structure type must initialize all instance fields of the type.
+
+- A structure type cannot inherit from other class or structure type and it cannot be the base of a class. However, a structure type can implement [interfaces](../keywords/interface.md).
+
+- You cannot declare a [finalizer](../../programming-guide/classes-and-structs/destructors.md) within a structure type.
+
+## Instantiation of a structure type
+
+In C#, you must initialize a declared variable before it can be used. Because a structure-type variable cannot be `null` (unless it's a variable of a [nullable value type](nullable-value-types.md)), you must instantiate an instance of the corresponding type. There are several ways to do that.
+
+Typically, you instantiate a structure type by calling an appropriate constructor with the [`new`](../operators/new-operator.md) operator. Every structure type has at least one constructor. That's an implicit parameterless constructor, which produces the [default value](default-values.md) of the type. You can also use the [default](../operators/default.md) operator or literal to produce the default value of a type.
+
+If all instance fields of a structure type are accessible, you can also instantiate it without the `new` operator. In that case you must initialize all instance fields before the first use of the instance. The following example shows how to do that:
+
+[!code-csharp[without new](~/samples/csharp/language-reference/builtin-types/StructType.cs#WithoutNew)]
+
+In the case of the [built-in value types](value-types.md#built-in-value-types), use the corresponding literals to specify a value of the type.
+
+## Passing structure-type variables by reference
```

Comment:
I intentionally don't use the words "reference semantics" here to avoid confusion between passing variables by reference and reference types. I see #4561 warns about that several times. By the way, thanks for the link to that issue. I've tried to update the article inline with the guidance there.

Reply:
I really like the way you've worded this.